src/lib/assetBridger/ethBridger.ts

```diff
@@ -115,22 +111,13 @@ export class EthBridger extends AssetBridger<
     await this.checkL1Network(params.l1Signer)
     await this.checkL2Network(params.l2Provider)
 
-    const gasEstimator = new L1ToL2MessageGasEstimator(params.l2Provider)
```

Comment:
Can the l2Provider now be removed from the params?

Reply:
True, removed